CharlieBackend.Core/Entities/Lesson.cs

```diff
@@ -24,6 +21,6 @@ public Lesson()
 
         public virtual Theme Theme { get; set; }
         
-        public virtual ICollection<Visit> Visits { get; set; }
+        public virtual IList<Visit> Visits { get; set; }
```

Comment:
Are you sure that it is a good solution?

Reply:
Ilist gives more possibilities foreach loop or a for loop to iterate